README.md

```diff
@@ -15,7 +15,9 @@ This project contains a todo list application.
 ### Setup
 
 #### `.env` requirements
-No `.env` requirements at this time.
+REACT_APP_API=the api url
```

Comment:
I would suggest in your code (when this is being used) that you have this or what link you were provided in to use, because not everyone will have a backend written as well